### DISCUSSION TITLE
Are you looking Opportunities 🤓 Open for Hiring?

### DISCUSSION THREAD

**Discussion by AlexisHNortal** [community/community#135749] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi Git Community,
 this i not quite a discussion i would like to invite all to Review Job Opportunities in my repositories

**Comment by LiteBrite82** [comment]
Hi @AlexisHNortal , we're happy you're here 😀

However, we currently do not allow job listings on the Community Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
